tests/fv/test_multi_host.py

```diff
@@ -0,0 +1,62 @@
+from time import sleep
```

Comment:
The multi host test as you've written it is very much an implementation of the IPv4 version of the demo.  It is probably worth stating that.